src/handlers/cell.ts

```diff
@@ -302,6 +314,16 @@ export namespace CellManager {
       editor.editor.removeLineClass(line, 'wrap', LINE_HIGHLIGHT_CLASS);
     });
   }
+
+  /**
+   * Remove line numbers and all gutters from cell.
+   * @param cell The cell to cleanup.
+   */
+
+  export function setOffOptions(cell: Cell) {
+    const editor = cell.editor as CodeMirrorEditor;
+    editor.setOption('lineNumbers', false);
```

Comment:
I haven't tested locally yet, but does that mean the line numbers will keep switching between the on and off states?

This was changed in https://github.com/jupyterlab/debugger/pull/114 to fix https://github.com/jupyterlab/debugger/issues/106.

Or a screencast could be helpful here to see what it looks like.

Reply:
This setOffOptions is executed only when Debbugger widget is closed.
![example](https://user-images.githubusercontent.com/34033913/69233158-b64c4480-0b8c-11ea-9d3c-0891391ef51e.gif)

Reply:
But line numbers could already be enabled before using the debugger:

![image](https://user-images.githubusercontent.com/591645/69234513-a1bd7b80-0b8f-11ea-9346-d0eac9c7be54.png)

Wondering how that would play together.

Reply:
You mean that user set by default that or just set by default by us ?

Reply:
Yes by a user. It's also possible to set it via the settings editor:

![image](https://user-images.githubusercontent.com/591645/69235778-738d6b00-0b92-11ea-9128-7931f37ae60b.png)

Reply:
ok. i will just implement clearAllGutters